app/controllers/items_controller.rb

```diff
@@ -1,8 +1,8 @@
 class ItemsController < ApplicationController
-  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
+  before_action :authenticate_user!, only: [:edit, :update, :destroy]
   
   def index
-    @items=Item.where("status = 'approved'")
+    @items=Item.where(status: 'approved')
```

Comment:
space around the `=`

Reply:
Spacing fixed throughout items_controller.